massgov/mayflower
# DP-11651  address contact sidebar

## Description
- This PR adds in conditional logic to hide the contact information block in the sidebar of a How To page if there is only an Address given.
- To display this difference, I have created a howTo AddressOnly page within patternlab

## Related Issue / Ticket

- JIRA: DP-11651

## Steps to Test
- Go Here: https://mayflower.digital.mass.gov/b/feature/DP-11651--address-contact-sidebar/patterns/05-pages-howto-addressOnly/05-pages-howto-addressOnly.html?1558703962100
- You should see: the contact bar on the right hand side hidden, showing how a page would look if there was only an address entered for contact information. The address will show in the the Department of Unemployment Assistance accordion, only. 
- Compare this page to the general How To page to see the differences of having only an address vs having more contact information available.
